src/main/java/org/dependencytrack/tasks/VulnerabilityAnalysisTask.java

```diff
@@ -64,7 +64,7 @@ public class VulnerabilityAnalysisTask implements Subscriber {
     public void inform(final Event e) {
         if (e instanceof VulnerabilityAnalysisEvent) {
             final VulnerabilityAnalysisEvent event = (VulnerabilityAnalysisEvent) e;
-
+            LOGGER.info("Analyzing vulnerabilities");
```

Comment:
Same as for the policy evaluation task, this message by itself isn't helpful as it lacks context.